dev-tools/compose/data/provenance/config/genesis.json

```diff
@@ -184,6 +191,15 @@
               "amount": "12500000000000000000"
             }
           ]
+        },
+        {
+          "address": "tp1hjvympqheaer7v3ylr6csu6dfs8xq34evl3zg8",
```

Comment:
which public key is this associated with?

Reply:
I got an affiliate from Robert for smart key integration testing. I added it to the build.gradle for example-kotlin.
https://github.com/provenance-io/p8e-gradle-plugin/blob/6a1327d8626b723db50d264e73f2b24302fe8611/example-kotlin/build.gradle#L71

Reply:
Could we instead setup local5 to be smart key for us? Only because those 5 keys are already setup across a few different places. It would be easier to not worry about trying to add local6 in all those places.

Reply:
@wbaker-figure I wonder if we should rename that key in the example-kotlin build.gradle to like 'smartKey1' instead of 'local6' to make it more clear?

Reply:
Setup local 5 as the smart key for us and rename it smartKey1? Or just go with smartKey1?

Reply:
I would say rename the newly-added 'local6' to 'smartKey1'